Kitodo/src/main/java/de/sub/goobi/metadaten/RenderableEdit.java

```diff
@@ -40,19 +39,20 @@
 
     /**
      * Constructor. Creates a RenderableEdit.
-     * 
-     * @param metadataType
+     *
+     * @param metadataTypeInterface
      *            metadata type editable by this drop-down list
      * @param binding
      *            metadata group that shall instantly be updated if a setter is
      *            invoked
      * @param container
      *            metadata group this drop-down list is showing in
      */
-    public RenderableEdit(MetadataType metadataType, MetadataGroup binding, RenderableMetadataGroup container) {
-        super(metadataType, binding, container);
+    public RenderableEdit(MetadataTypeInterface metadataTypeInterface, MetadataGroupInterface binding,
```

Comment:
You added "Interace" to the variable name like commented by @Beacze many times above and below.

Reply:
These changes were introduced by the refactoring tools. (Normally, this is the wanted behavior: If you rename a class, the corresponding variable name should be renamed.) I had already changed a lot of them back after Beatrycze’s remark, but I still had missed a lot of. The problem is that I didn’t find a good way to search for them yet. Just searching the code for `Interface` (even case-sensitive) is way to loose.